java-manta-it/src/test/resources/testng-it.xml

```diff
@@ -11,65 +12,190 @@
             <class name="com.joyent.test.util.RandomInputStreamTest"/>
         </classes>
     </test>
-
+    <test name="Manta Client Directory Tests">
```

Comment:
In the below changes, we moved from wildcard test definition to explicit test definition. This will make it really easy for engineers to forget to add a test to this file after they create it. Is there a way for us to define the tests more dynamically?

Reply:
We might have to use `beanshell` scripts which will be defined in the `testing-it.xml` file.

References: 
- [Test-NG Docs](https://testng.org/doc/documentation-main.html#beanshell).
- Blog [post](https://rationaleemotions.wordpress.com/2013/08/21/beanshell-and-testng/) that I found online.

Reply:
Let's leave it as is, but please add a section to the documentation calling out that the tests must be added to the XML files.

Reply:
I think for Chris to run the `nightly` tests for the `Java SDK` in **Jenkins**. It would be preferable and much convenient if he is to run the test group from the command line , for eg 
`mvn clean test -DgroupToRun=nightly`
rather than figuring it out how to invoke them using `TestNG` classes.

Reply:
> Let's leave it as is, but please add a section to the documentation calling out that the tests must be added to the XML files.

Testing Documentation updated with the latest [commit](https://github.com/joyent/java-manta/pull/507/commits/c3638ac33ddeafc330cd336c0252cc121fef8c19).